### PR TITLE
feature(pu): add ez/mz/az pytorch ddp support

### PR DESCRIPTION
**Pull request by puyuan1996**
- add ez/mz/az pytorch ddp support

### CODE REVIEW

File: lzero/config/utils.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import numpy as np
+import torch.distributed as dist
+from ding.utils.default_helper import error_wrapper
+from easydict import EasyDict
+
+
+def get_world_size() -> int:
```
Comment (PaParaZz1) [inline lzero/config/utils.py:7]: why not import this from `ding/utils`

File: lzero/policy/efficientzero.py [hunk 2]
```diff
@@ -473,6 +475,8 @@ def _forward_learn(self, data: torch.Tensor) -> Dict[str, Union[float, int]]:
         total_grad_norm_before_clip = torch.nn.utils.clip_grad_norm_(
             self._learn_model.parameters(), self._cfg.grad_clip_value
         )
+        if self._cfg.multi_gpu:
```
Comment (PaParaZz1) [inline lzero/policy/efficientzero.py:478]: You should sync gradient first, then do grad clip

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: lzero/config/utils.py (hunk 1 above)
Comment (puyuan1996) [reply lzero/config/utils.py:7]: ok

File: lzero/policy/efficientzero.py (hunk 2 above)
Comment (puyuan1996) [reply lzero/policy/efficientzero.py:478]: ok

**Review by PaParaZz1 — APPROVED** [review]
